cogs/fun.py

```diff
@@ -90,10 +90,14 @@ async def quote(self, ctx: commands.Context, character: str = None):
         await ctx.send(embed=embed)
 
     @commands.command(description='Calculate if you and your crush will work out.',
-                      help='Yuuto mastered the art of shipping users and can now calculate if you and your crush will '
-                           'work out.',
+                      help='Yuuto mastered the art of shipping users and can now calculate if you and your crush will work out.',
+                      usage='<user_1> <user_2>',
```

Comment:
I would make the parameters not optional instead and handle when they are not supplied (this can be done in a later change tho)